### DISCUSSION TITLE
[Snapcast, Spotify] Stream often stops at song end (but not always), and song plays "past elapsed time"??

### DISCUSSION THREAD

**Discussion by philhzss** [music-assistant/support#2970] [Q&A] [answered]
Hi all,

I'm new here, searched around, checked the troubleshooting page etc. I'm almost sure it's something I'm doing wrong on my side so I don't want to pollute the repo with an issue, but thought I'd ask here.

If you take a look at my screenshot, it shows the song has been playing for 23 mins even though it's only 6 mins long. The music will be streaming fine, sometimes it can run for an hour without issue, yet sometimes this happen every 2 songs, but eventually, the stream suddenly stops, exactly when the next song should start.

And when this happens, I check Music Assistant and I see what I'm showing in the screenshots, the "elapsed time" is longer than the total song length, it is still counting up but it's just silence. Pressing rewind or forward to skip to another song, or pausing/unpausing solves the issue and music restarts instantly.

I only set this all up yesterday, it's my first time with Snapcast and Music Assistant, so chances are I'm doing something wrong?? I'm streaming to 2 devices but it seems to be affecting the stream no matter what devices I'm streaming to.. I tried increasing the buffer size on my snapserver, the chunk size too, didn't change anything. I'm using the external Snapserver (self installed with docker) provider if that matters. Has anyone ever seen this? I've only really tested it with Spotify. It's the only provider I have. I do have a local file test song directly setup in my Snapserver config, and that one seems to loop perfectly without issue (but its just one song).

So I don't know if something is wrong between Spotify and Music Assistant, or Music Assistant and Snapcast, or something else. Please let me know if you have any ideas! For the screenshot I stopped after 23 mins but I tried letting it run earlier and it was all the way up to 2h, still counting up, silence. Pause and unpause fixed it instantly.

Thank you!

![image](https://github.com/user-attachments/assets/f55614cb-32bb-4a07-bef2-69f4ba1e58b8)

**Comment by OzGav** [comment, marked as answer]
Probably this https://github.com/music-assistant/support/issues/2863

Please monitor that issue and if it is resolved and you still have problems then ask again.

**Reply by philhzss** [reply]
Ah okay. Sorry I missed that. Thank you

**Comment by groldo** [comment]
Hi there,
I'm still experiencing this problem using the latest server release `2.3.6`.
Furthermore the referenced issue is now leading to a 404.
![music-assistant](https://github.com/user-attachments/assets/85f0c2e8-3c7a-44ae-817b-06ca3b40aacb)

**Reply by OzGav** [reply]
We have identified a problem with ffmpeg. An update has been shipped in 2.4.0b14.

issue link updated

**Reply by groldo** [reply]
I'm using `2.4.0b15` now for some time and the issue seems to be solved.

Thank you!
